src/js/utils/test/colors.test.js

```diff
@@ -0,0 +1,14 @@
+const assert = require('assert');
+const colors = require('../colors');
+
+describe('utils.colors', () => {
+  it('should return #aaabac for RGB()', () => {
+    assert.equal(colors.getColor('rgb(170, 171, 172)', '#aaabac'));
+  });
+  it('should return #ff5858 for the named color red', () => {
+    assert.equal(colors.getColor('red', '#ff5858d'));
+  });
+  it('should return #1a5c29 for the hex color #1a5c29', () => {
+    assert.equal(colors.getColor('#1a5c29', '#1a5c29'));
+  });
+});
```

Comment:
```suggestion
const assert = require('assert');
const colors = require('../colors');

describe('utils.colors', () => {
  it('should return #aaabac for RGB()', () => {
    assert.equal(colors.getColor('rgb(170, 171, 172)'), '#aaabac');
  });
  it('should return #ff5858 for the named color red', () => {
    assert.equal(colors.getColor('red'), '#ff5858');
  });
  it('should return #1a5c29 for the hex color #1a5c29', () => {
    assert.equal(colors.getColor('#1a5c29'), '#1a5c29');
  });
});
```

Fix the syntax for assert

Reply:
Thx, my bad ... didn't notice that ... i'll be on it asap